comparisontool/templates/comparisontool/technote.html

```diff
@@ -113,32 +113,36 @@ <h2>GI Bill&reg; benefits calculator</h2>
+                <p>Percent of monthly salary: The pie chart shows you how much of your anticipated post-graduation income will be spent repaying your student loans. Since institution-specific salary data is not available, this tool uses the average salary one year after graduation for a student with a bachelor’s degree, as reported by the <a class="exit-link" href="http://nces.ed.gov/surveys/b&amp;b/">Department of Education’s Baccalaureate and Beyond</a> study. Adjusted for inflation, this figure equates to $30,922 per year on a pre-tax basis (a monthly salary of $2,577).</p>
+                
                 <h3>Debt burden</h3>
-                <p>Your debt burden measures your estimated debt after school against the national average starting salary for college grads. Again, it’s an average. If you make more money, you’ll have a lower burden. It also doesn’t factor in specific information about your school or special repayment programs, like Income Based Repayment or Public Service Loan Forgiveness.</p>
-                <p>Total debt is calculated as described above. Since institution-specific salary data is not available, this version of the tool uses the average salary one year after graduation for a student with a bachelor’s degree, as reported by the <a class="exit-link" href="http://nces.ed.gov/surveys/b&amp;b/">Department of Education’s Baccalaureate and Beyond</a> study. Adjusted for inflation, this figure equates to $30,922 per year on a pre-tax basis (a monthly salary of $2,577).</p>
-                <p>Debt burden does not take into account that the average starting salary of graduates of particular institutions may vary. This tool will provide an indication of whether debt burden is low, medium, or high as follows:
+                <p>Your debt burden measures your estimated debt after school against the national average starting salary for college grads. Again, it’s an average. If you make more money, your debt will be a lower percentage of your income. It also doesn’t factor in specific information about your school or special repayment programs, such as Income-Driven Repayment or Public Service Loan Forgiveness.</p>
+
+                <p>Total debt is calculated as described above. Since institution-specific salary data is not available, this tool uses the average salary one year after graduation for a student with a bachelor’s degree, as reported by the <a class="exit-link" href="http://nces.ed.gov/surveys/b&amp;b/">Department of Education’s Baccalaureate and Beyond</a> study. Adjusted for inflation, this figure equates to $30,922 per year on a pre-tax basis (a monthly salary of $2,577).</p>
```

Comment:
`http://nces.ed.gov/surveys/b&amp;b/` doesn't appear to be a valid URL?

Reply:
Thanks, Ans! Fixed and pushed!